explain-overview.md

```diff
@@ -57,6 +57,79 @@ The following describes the output of the `EXPLAIN` statement above:
+> |   ├─Selection_10(Build)         | 1.25     | cop[tikv] |                       | not(isnull(test.t2.a))                                                                                          |
+> |   │ └─IndexRangeScan_8          | 1.25     | cop[tikv] | table:t2, index:ia(a) | range: decided by [eq(test.t2.a, test.t1.a)], keep order:false, stats:pseudo                                    |
+> |   └─TableRowIDScan_9(Probe)     | 1.25     | cop[tikv] | table:t2              | keep order:false, stats:pseudo                                                                                  |
+> +---------------------------------+----------+-----------+-----------------------+-----------------------------------------------------------------------------------------------------------------+
+> +---------------------------------+----------+-----------+-----------------------+------------------------------------------------------------------------------+
+> | id                              | estRows  | task      | access object         | operator info                                                                |
+> +---------------------------------+----------+-----------+-----------------------+------------------------------------------------------------------------------+
+> | Projection_12                   | 10000.00 | root      |                       | test.t2.a                                                                    |
+> | └─Apply_14                      | 10000.00 | root      |                       | CARTESIAN left outer join                                                    |
+> |   ├─TableReader_16(Build)       | 10000.00 | root      |                       | data:TableFullScan_15                                                        |
+> |   │ └─TableFullScan_15          | 10000.00 | cop[tikv] | table:t1              | keep order:false, stats:pseudo                                               |
+> |   └─Limit_17(Probe)             | 1.00     | root      |                       | offset:0, count:1                                                            |
+> |     └─IndexReader_21            | 1.00     | root      |                       | index:Limit_20                                                               |
+> |       └─Limit_20                | 1.00     | cop[tikv] |                       | offset:0, count:1                                                            |
+> |         └─IndexRangeScan_19     | 1.00     | cop[tikv] | table:t2, index:ia(a) | range: decided by [eq(test.t2.a, test.t1.b)], keep order:false, stats:pseudo |
+> +---------------------------------+----------+-----------+-----------------------+------------------------------------------------------------------------------+
+> 
+> -- From v6.4.0:
+>
+> -- You can find the difference between `estRows` of `IndexLookUp_11`、`Selection_10`、`IndexRangeScan_8` and `TableRowIDScan_9` and that before v6.4.0.
```

Comment:
```suggestion
> -- You can find that the `estRows` column values for `IndexLookUp_11`, `Selection_10`, `IndexRangeScan_8`, and `TableRowIDScan_9` since v6.4.0 is different from that before v6.4.0.
```